Kernel/API/KeyCode.h

```diff
@@ -141,11 +172,11 @@ enum KeyModifier {
 
 struct KeyEvent {
     KeyCode key { Key_Invalid };
-    u32 scancode { 0 };
+    u8 map_entry_index { 0 };
+    u64 scancode { 0 };
```

Comment:
It looks like nothing really uses `scancode` in the userland after this PR, so we should rather remove it. That would prevent the potential confusion between between `map_entry_index` and `scancode` (the difference is not obvious to me, at least...)

Reply:
This is not easily removable as the kernel uses this attribute as well. Maybe rename it to "raw_scancode" will be more appropriate and `map_entry_index` => `layout_map_entry_index`?